dataland-backend/backendOpenApi.json

```diff
@@ -2847,6 +2847,16 @@
         }
       },
       "LksgGeneral": {
+        "required": [
+          "dataDate",
```

Comment:
The bar with the button should stick to the end of the form if one scrolls down to the footer.

Reply:
The bar with the button doesn't exist anymore. Now instead we have a SubmitSideBar that contains the submit button, Submit Message and links. As described by @fsoffel, this bar can scroll into the footer. However, since the browser window needs to be put into an unreasonably low height for this to actually matter I'd leave this issue unresolved.

Reply:
ok